introduction/from_flat_php_to_symfony2.rst

```diff
@@ -370,7 +370,7 @@ on the requested URI::
         echo '<html><body><h1>Page Not Found</h1></body></html>';
     }
 
-For organization, both controllers (formerly ``index.php`` and ``show.php``)
+For organization, both controllers (formerly ``/index.php`` and ``/index.php/show``)
```

Comment:
It seems I got confused by this line https://github.com/symfony/symfony-docs/pull/7838/files#diff-9bc9c246d5d01be55bf80517de478b73R333, but I understand now it does not need to be changed 😅